### ISSUE TITLE
Typo: `import definitelyNotAHuman` and other confusion in the readme?

### ISSUE THREAD

**Issue by ggorlen** [dougwithseismic/npm-definitely-not-a-robot#2]
From the readme:

```js
import definitelyNotAHuman from 'path-to-definitely-not-a-robot'

const browser = new definitelyNotAHuman()
```

Why would we want to call this variable "definitely not a human" (i.e., a robot)? Shouldn't this be `import definitelyNotARobot from 'path-to-definitely-not-a-robot'`?

Also, `path-to-definitely-not-a-robot` is usually the actual package name.

---

I realize that

> Be a human pretending to be a robot being a human robot human-being

is an attempt at humor, but it's very confusing. I suggest just stating what the library does simply.

**Comment by eng3** [comment]
Does the name matter?
Regardless, more detailed instructions would be helpful for beginners like myself.  It sounds like you are extending puppeteer so this is meant to drop in to replace it.

**Comment by ggorlen** [comment]
@eng3 Yes, it matters. I'm not extending Puppeteer, I'm just trying to answer [your stack overflow question](https://stackoverflow.com/questions/78148272/how-can-i-use-typescript-code-in-an-existing-node-project), but I wasn't able to easily determine what the library does or understand how to install/import it.

Beyond the name, the convention is to capitalize constructors, so I'd suggest `new DefinitelyNotARobot()`.

~~Now that you mention it, extending Puppeteer is the correct way to design a package like this. Hiding `browser.launch()` as this library does makes it more or less unusable, because being able to specify launch options is a critical piece of functionality that needs to be exposed.~~ It seems to be--my mistake.

**Comment by dougwithseismic** [comment]
Fwiw I think,  but can't remember, that I made it possible to swap your own
Puppeteer instance in - If I didn't then I definitely should have.

Again, I'll get to this next week - apologies for the delay, I'm without a
machine ✌️

**Comment by eng3** [comment]
@ggorlen I wasnt sure how to install/import it either hence why I asked the question.  As for the broader question of why anyone needs this package at all, I don't know.  I'm running out of ideas thus thought something like this was worth a try.  I didn't want to list the site by name in order to attempt to keep some anonymity due to the nature of this type of thing
